mypyc/test-data/run-strings.test

```diff
@@ -518,6 +518,18 @@ def test_format_method_python_doc() -> None:
 
 def test_chr() -> None:
     assert chr(57344) == '\ue000'
+    assert chr(0) == '\x00'
+    try:
+        chr(-1)
+        assert False
+    except ValueError:
+        pass
+    try:
+        chr(1114112)
+        assert False
+    except ValueError:
+        pass
+    assert chr(1114111) == '\U0010ffff'
```

Comment:
Also test `chr(x)` when `x` is not a compile time constant (e.g. `chr(100 + int())`.